okhttp/src/main/kotlin/okhttp3/Response.kt

```diff
@@ -162,8 +162,16 @@ class Response internal constructor(
 
   fun headers(name: String): List<String> = headers.values(name)
 
-  @JvmOverloads
-  fun header(name: String, defaultValue: String? = null): String? = headers[name] ?: defaultValue
+  fun header(name: String): String? = headers[name]
+
+  fun header(name: String, defaultValue: String): String = header(name) ?: defaultValue
+
+  @JvmName("-deprecated_header")
+  @Deprecated(
```

Comment:
I could be misunderstanding the experience for java and kotlin callers with source compatibility.  But this seems like effectively a new method for most people, so weird that it is deprecated.